org.eventb.emf.persistence.tests/pom.xml

```diff
@@ -7,7 +20,7 @@
     <!-- Name of the release engineering artifact -->
     <artifactId>releng</artifactId>
     <!-- REQUIRED: Version number of the release engineering artifact -->
-    <version>6.1.0.release</version>
+    <version>6.2.0-SNAPSHOT</version>
```

Comment:
This should be 7.0.0 as the update to the latest POM release engineering.

Reply:
oops a daisey